pages/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+var adj = ['astonishing', 'amazing', 'awesome', 'stunning', 'marvelous', 'spectacular', 'breathtaking', 'worldchanging']
+
+const Lista = (props) => (
+  <div>
+    <ul>
+      { props.ideas.map((idea, index) => <li key={ index }>{ idea }</li>) }
+    </ul>
+  </div>
+)
+
+class Index extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {ideas: []}
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+  handleSubmit(e) {
+    var ideas = this.state.ideas.slice()
```

Comment:
usá `const`, es buena práctica ya que refuerza la [inmutabilidad](https://medium.com/@brooklynunited/the-case-for-immutability-d4e45029b8bc), el programa no está cambiando nada realmente, es solo un mindset (`const` te explota si cambiás el valor de la variable)